src/System.Collections.Concurrent/src/System/Collections/Concurrent/ConcurrentQueue.cs

```diff
@@ -912,10 +912,6 @@ internal int High
     /// </summary>
     struct VolatileBool
     {
-        public VolatileBool(bool value)
```

Comment:
Interesting they chose this route instead of `bool[]` with [`Volatile.Read(ref bool)`](https://msdn.microsoft.com/en-us/library/gg712942.aspx) and [`Volatile.Write(ref bool, bool)`](https://msdn.microsoft.com/en-us/library/gg712815.aspx).